### ISSUE TITLE
Improve the sidebar navigation

### ISSUE THREAD

**Issue by bkjohnson** [department-of-veterans-affairs/va.gov-team#40066]
## Issue Description

The static content has a left sidenav component that doesn't seem consistent. Sometimes the user will see a sidenav like this:

![Simple sidenav on a facility page](https://user-images.githubusercontent.com/2008881/163650922-f7639509-c309-48d0-9dc0-e2b3acfc49b3.png)

(from [Palo Alto's Medical records office page](https://staging.va.gov/palo-alto-health-care/medical-records-office/index.html))

On other pages the user may see a sidenav like this:

![Different sidenav](https://user-images.githubusercontent.com/2008881/163651021-90f9e632-f01f-4369-82f6-4617805d689c.png)

(from the [eligibility page for burials & memorials](https://staging.va.gov/burials-memorials/eligibility/))

I think it would be good if the sidenav structure & behavior was consistent across all pages.

## Acceptance Criteria
- [ ] Decide on a single sidenav presentation style & structure

**Comment by bkjohnson** [comment]
:spaghetti: A bit of both, maybe? There's a [static pages template](https://github.com/department-of-veterans-affairs/content-build/blob/a455597e514e37bdf8c7f2d8633aee46d4758b47/src/site/navigation/sidebar_nav.drupal.liquid#L1) that's using the [`SideBarMenu` class from `formation`](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/3b63b10825f62f939a4053284c4bc61a96d0951c/packages/formation/js/sidenav.js#L140-L142). Then there's also the [_facilities_ sidebar template](https://github.com/department-of-veterans-affairs/content-build/blob/a455597e514e37bdf8c7f2d8633aee46d4758b47/src/site/navigation/facility_sidebar_nav.drupal.liquid#L22-L23) which relies on [some site-wide JS in `vets-website`](https://github.com/department-of-veterans-affairs/vets-website/blob/59dcb2fc8c26f6791f4ed8378a834528ec319941/src/platform/site-wide/side-nav/index.js#L13-L15).

I think there's an opportunity for a Design System sidebar/sidenav component in order to provide consistency.

**Comment by k80bowman** [comment]
Just to be clear, is your recommendation that we create a sidebar web component?

**Comment by bkjohnson** [comment]
Yes, unless there's some business or design reason why we have different sidebar components for different pages.

**Comment by k80bowman** [comment]
I know you brought up creating a sidebar web component the other day and I agree, I think it's a good idea. We already have the design, we could definitely prioritize building a web component.
